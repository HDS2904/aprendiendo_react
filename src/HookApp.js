import React from 'react'

export const HookApp = () => {
  return (
    <div>
      Hola Jonathan
			<li>pruebaaaaaaaaa</li>
    </div>
  )
}
