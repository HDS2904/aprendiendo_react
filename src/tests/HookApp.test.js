import React from 'react';
import { HookApp } from '../HookApp';
import '@testing-library/jest-dom/extend-expect';
const { render } = require("@testing-library/react")


describe('Pruebas en <HookApp/>', () => { 
	test('Debe mostrarse correctamente', () => { 
		const wrapper = render(<HookApp /> );
		expect(wrapper.container).toMatchSnapshot()
	})
})

// Nota: No se usara enzyme para esta secci√≥n ya que actualmente esta en desuso
// pues a pesar de crear adaptadores, el autor recomienda migrar por no poder
// continuar dandole soporte.