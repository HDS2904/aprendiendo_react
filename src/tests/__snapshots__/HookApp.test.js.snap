// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp/> Debe mostrarse correctamente 1`] = `
<div>
  <div>
    Hola Jonathan
    <li>
      pruebaaaaaaaaa
    </li>
  </div>
</div>
`;
